esp/esp/program/modules/forms/admincore.py

```diff
@@ -32,13 +33,21 @@ def __init__(self, program, *args, **kwargs):
     def load_data(self):
         lunch_timeslots = Event.objects.filter(meeting_times__parent_class__parent_program=self.program, meeting_times__parent_class__category__category='Lunch').distinct()
         self.initial['timeslots'] = lunch_timeslots.values_list('id', flat=True)
+        should_generate_constraints = True
+        if Tag.getTag('no_lunch_constraints'):
```

Comment:
Wait, so is this tag just the default value for the form?  That seems unnecessary; can we just keep things simple?  (Also, if we're keeping it, it should be a `getBooleanTag`.